include/deal.II/base/mpi_consensus_algorithms.h

```diff
@@ -797,6 +923,81 @@ namespace Utilities
       };
 
 
+#ifndef DOXYGEN
+      // Implementation of the functions in this namespace.
+
+      template <typename T1, typename T2>
+      std::vector<unsigned int>
```

Comment:
Shouldn't these be inline?

```suggestion
      inline std::vector<unsigned int>
```

Reply:
You don't technically need the `inline` statement as these are templated functions and thus automatically `inline` (i.e., not causing duplicate symbols during linking if they get included from multiple compilations units). But we tend to write `inline` in most occurrences. (But I would not stop the PR from this minor style issue, it has waited for long already.)